docs/dev/development.adoc

```diff
@@ -42,6 +42,7 @@ A PR must include:
 
  * Descriptive context that outlines what has been changed and why
  * A link to the active or open issue it fixes (if applicable)
+ * (OPTIONAL) A signed commit is good practice. You may sign your commit using `git commit -s` or `git commit --amend --no-edit -s` to a previously created commit
```

Comment:
👍

Reply:
Wouldn't it be more accurate to use "sign off" rather than just "sign"?  To make it at least a little bit clear that this is not GPG signing (`-S`)?